### PR TITLE
adding graphic placeholders for CYOA.

### PR DESCRIPTION
**Pull request by paul-grassi**

### CODE REVIEW

File: sp800-63a/sec5_proofing.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The following sections list the objectives and steps a CSP SHALL follow to ident
 
 ## <a name="validate"></a>5.2. Identity Evidence Validation
 
-The goal of identity validation is to collect from the applicant the most appropriate identity evidence (for example, a passport or drivers license) and determine its authenticity, validity, and accuracy.  Identity validation is made up of two process steps: collecting the appropriate identity evidence and confirming the data contained on the identity evidence is valid and related to an actual, live individual.
+The goal of identity validation is to collect from the applicant the most appropriate identity evidence (for example, a passport or drivers license) and determine its authenticity, validity, and accuracy.  Identity validation is made up of two process steps: collecting the appropriate identity evidence, confirming the evidence is genuine and authentic, and confirming the data contained on the identity evidence is valid, current, and related to an actual, live individual.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md:14]: Isn't that three process steps now?

I'd suggest using a verb other than "confirming" multiple times. Perhaps "validating the evidence" and "verifying the data..." to be consistent with GPG 45 terminology?

File: sp800-63a/sec5_proofing.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ Once identity evidence is obtained by the CSP, the accuracy, authenticity, and i
 
 ## 5.3. <a name="verify"></a> Identity Verification
 
-The goal of identity validation is to establish a linkage to the physical, live existence of the claimed identity to the person actually presenting the evidence.
+The goal of identity validation is to confirm and establish a linkage between the claimed identity to the to the physical, live existence of the person actually presenting the evidence.
```
Comment (jimfenton) [inline sp800-63a/sec5_proofing.md]: s/to the to the/and the/

**Review by jimfenton — CHANGES_REQUESTED** [review]
I notice the two new graphics aren't referenced anywhere. I gather these are just placeholders for now?

File: sp800-63a/sec5_proofing.md (hunk 1 above)
Comment (paul-grassi) [reply sp800-63a/sec5_proofing.md:14]: validation is not verification (linking it to the person presenting it).  Maybe: determining that the evidence is genuine and authentic, and confirming the data contained on the identity evidence is valid, current, and related to an actual, live individual.

I munge this as 2 steps.  Collect, then do all the validation.  Happy to make it 3.

File: sp800-63a/sec5_proofing.md (hunk 2 above)
Comment (paul-grassi) [reply sp800-63a/sec5_proofing.md]: good catch.  Can you fix that directly?

**Comment by paul-grassi** [comment]
Yup, graphics are just a placeholder.

File: sp800-63a/sec5_proofing.md (hunk 2 above)
Comment (jimfenton) [reply sp800-63a/sec5_proofing.md]: Sure. Do you want me to do anything with the other comment?

Comment (paul-grassi) [reply sp800-63a/sec5_proofing.md]: @jimfenton this looks really good.  It came across weird in email, hence my initial response, but online it looks great!

**Comment by paul-grassi** [comment]
Sorry about the confusion.  It was the way GH put things in email.  I am good with your edits.  Please merge if you are ok.

**Review by jimfenton — APPROVED** [review]

**Pull request merged**